easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.3.1-intel-2015a-eb-deps.eb

```diff
@@ -0,0 +1,52 @@
+name = 'OpenFOAM'
+version = '2.3.1'
+versionsuffix = '-eb-deps'
+paraversion = '4.3.1'
+pyver = '-Python-2.7.10'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package. 
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer, 
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://downloads.sourceforge.net/foam/%(version)s']
+sources = [
+    SOURCE_TGZ,
+    'ThirdParty-%(version)s.tgz',
+]
+
+# fix for cyclic symlink issue, which may cause unpacking to fail
+# see also http://www.openfoam.org/mantisbt/view.php?id=1191
+# fixed in recent versions: https://github.com/OpenFOAM/OpenFOAM-2.3.x/commit/f7a485069c778495cc39b308580289f6c2d47163
+unpack_options = "--exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/backgroundMeshDecomposition"
+unpack_options += " --exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/cellShapeControlMesh"
+
+patches = [
+    'cleanup-OpenFOAM-%(version)s.patch',
+    'OpenFOAM-2.3.0_libreadline.patch',
+    ('cleanup-ThirdParty-%(version)s.patch', ".."),  # patch should not be applied in OpenFOAM subdir
+    'external-3rd-OpenFOAM-%(version)s.patch',
+    ('external-3rd-ThirdParty-%(version)s.patch', ".."),
```

Comment:
because we have 2 software stack under `OpenFOAM`
Anyway, the the first patch made by you. ;)